modules/EnsEMBL/Web/Document/Image/R2R.pm

```diff
@@ -53,8 +53,14 @@ sub find_ss_in_compara {
 
     my $gma = $database->get_GeneMemberAdaptor();
     my $gta = $database->get_GeneTreeAdaptor();
+    my $gda = $database->get_GenomeDBAdaptor();
```

Comment:
$gda is also not being used elsewhere so can be removed.

Reply:
Likewise.